packages/fx-core/tests/plugins/solution/solution.arm.test.ts

```diff
@@ -194,7 +194,9 @@ module teamsFxConfig './config.bicep' = {
 }
 
 output provisionOutput object = provision
-output teamsFxConfigurationOutput object = teamsFxConfig`
+output teamsFxConfigurationOutput object = teamsFxConfig
+
```

Comment:
is content of the main.bicep changed by prettier automatically? maybe we should not have too many line breaks at the end (removing the line break at 198?)

Reply:
This string is used to compare with generated output file of project file.
As the generated file has 2 \n in the end. I added it here.